src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -308,6 +309,9 @@ def _new_job_cluster_metadata(self, new_cluster):
         self._data_security_mode = new_cluster.data_security_mode
         return []
 
+    def build_inherited_context(self, graph: DependencyGraph, child_path: Path) -> InheritedContext:
+        raise NotImplementedError("Should never get there!")
```

Comment:
do a bit better error message :) end user would be surprised and oncall won't understand what's happening

Reply:
done